styles/commit-preview-view.less

```diff
@@ -0,0 +1,20 @@
+@import "variables";
+
+.github-StubItem-git-commit-preview { // TODO Rename class
```

Comment:
It seems this class gets auto-generated? Will there be some sort of `CommitPreviewView`?

Reply:
That class is actually a temporary one that's applied to "stub items," which are used during deserialization for timing reasons. I've added a `github-CommitPreview-root` class to the `<div>` that should always be there 👌